### PR TITLE
[webui] Fix Style/FormatString Rubocop cop

### PR DESCRIPTION
**Pull request by eduardoj**

### CODE REVIEW

**Comment by eduardoj** [comment]
[Fix Style/FormatString Rubocop cop complaints](https://trello.com/c/j1OQESXS/443-fix-style-formatstring-rubocop-cop-complaints)

**Comment by eduardoj** [comment]
> @Ana06:
> why did you write `Style::FormatString` instead `Style/FormatString`?
> [...]

Changed to `Style/FormatString`.

File: src/api/app/controllers/webui/project_controller.rb [hunk 1]
```diff
@@ -982,7 +982,7 @@ def check_devel_package_status(currentpack, p)
     dproject = dp.project
     currentpack['develproject'] = dproject
     currentpack['develpackage'] = dp.name
-    key = '%s/%s' % [dproject, dp.name]
+    key = format('%s/%s', dproject, dp.name)
```
Comment (Ana06) [inline src/api/app/controllers/webui/project_controller.rb]: Can be changed by string interpolation :wink: 
``` Ruby
key = "#{dproject}/#{dp.name}"
```

File: src/api/app/jobs/project_status_calculator.rb [hunk 2]
```diff
@@ -125,7 +125,7 @@ def check_md5(packages)
 
   # parse the jobhistory and put the result in a format we can cache
   def parse_jobhistory(dname, repo, arch)
-    uri = '/build/%s/%s/%s/_jobhistory?code=lastfailures' % [CGI.escape(dname), CGI.escape(repo), arch]
+    uri = format('/build/%s/%s/%s/_jobhistory?code=lastfailures', CGI.escape(dname), CGI.escape(repo), arch)
```
Comment (Ana06) [inline src/api/app/jobs/project_status_calculator.rb]: Can be changed by string interpolation :wink:

File: src/api/app/models/issue.rb [hunk 3]
```diff
@@ -129,7 +129,7 @@ def render_axml
   end
 
   def to_axml(_opts = {})
-    Rails.cache.fetch('issue_%d' % id) do
+    Rails.cache.fetch(format('issue_%d', id)) do
```
Comment (Ana06) [inline src/api/app/models/issue.rb]: Can be changed by string interpolation :wink:

File: src/api/app/models/product.rb [hunk 4]
```diff
@@ -25,7 +25,7 @@ def self.all_products( project, expand = nil )
   end
 
   def to_axml(_opts = {})
-    Rails.cache.fetch('xml_product_%d' % id) do
+    Rails.cache.fetch(format('xml_product_%d', id)) do
```
Comment (Ana06) [inline src/api/app/models/product.rb]: Can be changed by string interpolation :wink:

File: src/api/app/models/package.rb [hunk 5]
```diff
@@ -835,7 +835,7 @@ def to_axml_id
   end
 
   def to_axml(_opts = {})
-    Rails.cache.fetch('xml_package_%d' % id) do
+    Rails.cache.fetch(format('xml_package_%d', id)) do
```
Comment (Ana06) [inline src/api/app/models/package.rb]: Can be changed by string interpolation :wink:

**Pull request merged**